server/main.ts

```diff
@@ -0,0 +1,67 @@
+/**
+ * Entry point script for Application server.
+ *
+ * Starts a server that receives requests from client/frontend, applying basic
+ * business logic making requests to the Rightfoot public API.
+ *
+ * Environment variables supported:
+ *  * `RIGHTFOOT_API_KEY`: Required. The API key used to authenticate with Rightfoot.
+ *  * `RIGHTFOOT_API_HOST`: The Rightfoot endpoint to make requests to.
+ *        Defaults to the Sandbox endpoint.
+ *  * `PORT`: The port for the server to listen on. Defaults to 8080.
+ */
+import express from 'express';
+import { RightfootApi } from 'rightfoot-node/1-3/api';
+import { Configuration } from 'rightfoot-node/1-3/configuration';
+
+import { App } from './app';
+import { ApiRouteHandlers } from './api-route-handlers';
+import { ApiRouter } from './api-router';
+
+/**
+ * Creates a Rightfoot API client from environment variables set.
+ */
+function getRightfootClient(): RightfootApi {
```

Comment:
Hmm this feels more fitting since we're not fetching
```suggestion
function createRightfootClient(): RightfootApi {
```
it's also more in line with `createAndStartApp`